src/fragrance_rater/services/kaggle_importer.py

```
@@ -0,0 +1,400 @@
+
+        Returns:
+            Dictionary mapping accord names to intensities (0-1).
+        """
+        if not accords_str:
+            return {}
+
+        accords: dict[str, float] = {}
+        parts = accords_str.split(",")
+
+        for i, part in enumerate(parts):
+            part = part.strip()
```

Ruff (0.15.12)

[warning] `for` loop variable `part` overwritten by assignment target

(PLW2901)